### ISSUE TITLE
Boxplot data

### ISSUE THREAD

**Issue by revenz** [apexcharts/Blazor-ApexCharts#113]
Trying to figure out the box plot chart, apexcharts expects [minimum, lowquartile, median, highquartile, maximum] array.  

But Blazor-ApexCharts, not sure how to pass that along.  

```
        <ApexChart TItem="BoxPlotData">
            <ApexBoxPlotSeries TItem="BoxPlotData"
                        Items="Data"
                        XValue="@(e => e.Name)"
                        YValue="@(e => new [] {e.Minimum, e.LowQuartile, e.Median, e.HighQuartile, e.Maximum})" />
        </ApexChart>
```
Thought it would be something like this, but obviously wrong.   How can I do this?

TIA

**Comment by joadan** [comment]
Hello,

Have you checked https://apexcharts.github.io/Blazor-ApexCharts/boxplot-charts 

You just have to provide the X-Y properties and we will automatically calculate the values.

**Comment by revenz** [comment]
yeah i saw that.  my database calculates these values at a given interval, dont want to read all the data from the db, then pass that to the client in a http request.
anyway I can just pass in those 5 values that apexchart expects?

**Comment by joadan** [comment]
Hello,

Sorry but I think I got this all wrong. 
Looking at the Apexchart spec it looks like the list should be the values like 

_The xy format accepts [{ x: category/date, y: [min, q1, median, q3, max] }]_
https://apexcharts.com/docs/chart-types/boxplot/ 

I will look into this asap it's a priority

**Comment by joadan** [comment]
Fixed in v0.9.13-beta
